### DISCUSSION TITLE
Smartphone HONOR 8 (android8) Xdrip does not work

### DISCUSSION THREAD

**Discussion by marcinkowski11** [NightscoutFoundation/xDrip#2851] [General]
Hi
 I have a new smartphone from year 2016 it is a Honor 8 (Huawei) Itis Banned from android service. have no google playstore etc. It has Emui 8 and android 8. I tried to install xdrip on it. the latest one from may 2023. After installing I see the agreement  text. than I confirm it and after it my phone goes back to homescreen. when I activate xdrip again. the screen gets black for few seconds and than back to homescreen.  I have no Idea what the problem is

**Comment by Navid200** [comment]
Thanks for posting here.  

When you open xDrip for the very first time after install, you will see two different notes that require you to acknowledge.  I am showing both below.    
  
Did you get both?  

![Screenshot_20230525-134807](https://github.com/NightscoutFoundation/xDrip/assets/51497406/c2e87200-d2ab-45ad-b0cb-a3ee22833894)
![Screenshot_20230525-134718](https://github.com/NightscoutFoundation/xDrip/assets/51497406/d17f1254-1fd3-43fa-8eaf-ec0771699b9e)

**Reply by marcinkowski11** [reply]
Exactly. This is what I can see. After I confirm it, the screen gets black and than it moves back to homescreen

**Reply by Navid200** [reply]
Do you have a PC or laptop?  Do you mind installing Android Studio on it?
Then, you can enable developer options on your phone, and attach it to the laptop using a USB cable.
Then, with Android studio running, if you look at logact in it, and repeat the process, you can capture the logs for when the problem occurs.

Then, looking at the logs, we may be able to figure out what is the cause of the problem.

**Reply by marcinkowski11** [reply]
I gave up with the phone HONOR EMUI seems not to work with xdrip. Thanks for support. I got an alternative phone
